pkgs/applications/science/biology/seaview/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, fetchurl, pkgs }:
+
+stdenv.mkDerivation rec {
+  version = "4.7";
+  name = "seaview-${version}";
+
+  src = fetchurl {
+    url = "ftp://pbil.univ-lyon1.fr/pub/mol_phylogeny/seaview/archive/seaview_${version}.tar.gz";
+    sha256 = "0fhyq7dcn0izhwcfin9ajsr7kmmsqm9f1np1rmhzg4digfwqb29n";
+  };
+
+  buildInputs = with pkgs; [ fltk libjpeg ];
+
+  patchPhase = "sed -i 's#PATH=/bin:/usr/bin#/usr/bin/env#' seaview.cxx";
```

Comment:
Am I reading the source correctly that this is only used to get `rm`? And in that case, can you patch in the actual path from coreutils instead?

Reply:
That's right it is just a single occurrence and only used to get `rm` working. In the next iteration I will follow your suggestion.